circle.yml

```diff
@@ -6,6 +6,7 @@ machine:
 dependencies:
   pre:
     - yarn global add typedoc
+    # chrome
```

Comment:
I'm using the Swagger UI which generates the interactive docs on the fly from a yml file.  https://swagger.io/swagger-ui/